.travis.yml

```diff
@@ -1,9 +1,5 @@
 language: python
 
-branches:
-    only:
-        - master
-
```

Comment:
was this preemptively set in the first place? or was it set because of an actual problem? If the first, then maybe this should be its own PR.